R/helper_patient_data_fix.R

```diff
@@ -564,6 +564,17 @@ fix_id <- function(id) {
 }
 
 
+#### insulin regimen ####
+
+extract_regimen <- function(raw_input) {
+    output <- sub("^.*basal.*$", "Basal-bolus (MDI)", raw_input, ignore.case = TRUE)
```

Comment:
does this actually need the ".*" part if you are only looking for basal here? is just "basal" not enough for this case?

Reply:
I need everything, because the match is replaced, so I have to make sure that the entire string ist matched when it contains "basal" or something different in the other cases.